mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+python_version = 3.7
+show_error_context = true
+show_column_numbers = true
+disallow_untyped_defs = true
+disallow_incomplete_defs = true
+disallow_untyped_calls = true
+disallow_untyped_decorators = true
+disallow_subclassing_any = true
+warn_unused_ignores = true
+warn_unreachable = true
+
```

Comment:
Is it useful to enable these?
show_error_codes=True
pretty=True

Reply:
Wouldn't hurt, I'll add them.